src/utils/user.authentication.ts

```diff
@@ -18,11 +20,21 @@ export async function getAccessTokenForUser(
   userRepository: UserRepository,
   credentials: Credentials,
 ): Promise<string> {
+  console.log('getAccessTokenForUser', credentials.password);
   const foundUser = await userRepository.findOne({
-    where: {email: credentials.email, password: credentials.password},
+    where: {email: credentials.email},
   });
   if (!foundUser) {
-    throw new HttpErrors.Unauthorized('Wrong credentials!');
+    throw new HttpErrors.Unauthorized(
```

Comment:
Should we throw a 404 here or would that be a security risk?

Reply:
@b-admike Good point, I think 404 makes more sense.